pulsar-broker/src/main/java/com/yahoo/pulsar/broker/cache/LocalZooKeeperCacheService.java

```diff
@@ -123,32 +140,80 @@ private void initZK() throws PulsarServerException {
      *            if true copy policies from global zk to local zk else create a new znode with empty {@link Policies}
      * @throws Exception
      */
-    public void createPolicies(String path, boolean readFromGlobal) throws Exception {
+    @SuppressWarnings("deprecation")
+    public CompletableFuture<Optional<LocalPolicies>> createPolicies(String path, boolean readFromGlobal) {
+        checkNotNull(path, "path can't be null");
```

Comment:
In this case is more of a sanity check.. we could even remove the check alltogether